diff --git a/sources/source_base.js b/sources/source_two_methods_merged.js
index deadbeef..214a0dd 100644
--- a/sources/source_base.js
+++ b/sources/source_two_methods_merged.js
@@ -17,11 +17,6 @@ Object.assign( Foo.prototype, {
 		method1_call1();
 		method1_call2();
 		method1_call3();
-
-	},
-
-	method2: function () {
-
 		method2_call1();
 		method2_call2();
 
diff --git a/sources/source_base.js b/sources/source_two_methods_modified_cosmetics.js
index deadbeef..649e3e7 100644
--- a/sources/source_base.js
+++ b/sources/source_two_methods_modified_cosmetics.js
@@ -15,7 +15,7 @@ Object.assign( Foo.prototype, {
 	method1: function ( a, b ) {
 
 		method1_call1();
+		method1_call2(); // foo
 		method1_call3();
 
 	},
@@ -24,6 +23,6 @@ Object.assign( Foo.prototype, {
 
 		method2_call1();
+		method2_call2(); // bar
 
 	},
 
